spec/models/manageiq/providers/amazon/cloud_manager/vcr_specs/refresher_inventory_object_spec.rb

```diff
@@ -63,15 +63,6 @@
     end
   end
 
-  def table_counts_from_api
```

Comment:
Actually `table_counts_from_api` aren't called from anywhere else anymore so we don't need to override this, just moved the service_* counts into the base method https://github.com/ManageIQ/manageiq-providers-amazon/pull/748/files#diff-054ae1b3080de7c9ba1d8a11342cfa4d41b35def7a12a2cf8e1ebaef5d5604b0R169-R171